ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -182,6 +193,29 @@ tflite_setInputDim (const GstTensorFilterProperties * prop, void **private_data,
   return tflite_core_setInputDim (tf->tflite_private_data, in_info, out_info);
 }
 
+/**
+ * @brief The optional callback for GstTensorFilterFramework
+ * @param prop property of tensor_filter instance
+ * @param private_data : tensorflow lite plugin's private data
+ * @return 0 if OK. non-zero if error.
+ */
+static int
+tflite_reloadModel (const GstTensorFilterProperties * prop, void **private_data)
+{
+  tflite_data *tf;
+  tf = *private_data;
+  g_assert (*private_data);
+
+  if (prop->num_models != 1)
+    return -1;
+
+  /* if the model path is the same, skip reload */
+  if (tflite_core_compareModelPath (tf->tflite_private_data, prop->model_files[0]))
+    return 0;
```

Comment:
What if the weight values are updated with the given model?

Reply:
Right. we can provide the same model path with different weights. I'll remove this line.